config/routes.rb

```diff
@@ -33,6 +35,7 @@
   resources :limber_plates, controller: :plates do
     resources :children, controller: :plate_creation
     resources :tubes, controller: :tube_creation
+    resources :tube_racks, controller: :tube_rack_creation
```

Comment:
Don't quite understand what these routes are for

Reply:
Nope nor me. I think it's to do with asset creation, parent plate here with possible child creation routes?